testing/Makefile

```
@@ -0,0 +1,33 @@
+STACK_VERSION ?= 8.6.1
+CURRENT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+CLUSTER_DIGEST_FILE := cluster-digest.yml
+CLUSTER_INFO_FILE := cluster-info.json
+CLUSTER_NAME := e2e-elasic-agent-$(shell date +%s)
+SLACK_CHANNEL := @U0396725PSN
+
```

Use env var that can be redefined from the command line for the template and the output file

```suggestion
TEMPLATE_FILE ?= $(CURDIR)/test-cluster.yml.tpl
OUTPUT_FILE ?= $(CURDIR)/$(CLUSTER_INFO_FILE)
```